src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -442,32 +421,7 @@ private static string GetProjectIdentifier(string projectKey, string projectBran
         private static string Escape(string format, params string[] args) =>
             string.Format(CultureInfo.InvariantCulture, format, args.Select(WebUtility.UrlEncode).ToArray());
```

Comment:
I've now noticed that `args` is `string[]`. So there's really nothing that the culcure could do about those. Let's remove it
```suggestion
            string.Format(format, args.Select(WebUtility.UrlEncode).ToArray());
```